### DISCUSSION TITLE
Is there a way to stop this overwrite video ?

### DISCUSSION THREAD

**Discussion by mr-qansuh** [staxrip/staxrip#1248] [General]
Is there a way to stop this overwrite video
when i encode video like messi.mp4
i encode to 1080p then 720p then 480p by file patch when stax finish encoding 1080 and after finsih from 720 i found that extract video has come 720 Similarly with 480.

Is there a way to cancel over come?
and make stax in every encoding ad _new 
720p = _new_new
480p= _new_new_new

or fi i can add 1080 to end of file name Similarly with 720 and 480.

**Comment by jessielw** [comment]
Just change the name for each job in the top right

**Reply by mr-qansuh** [reply]
i use file patch for some video all in one jop

**Reply by jessielw** [reply]
> i use file patch for some video all in one jop

I'm not really sure what you mean. Do you mean file batch?  

Just set up a 480p, 720p, and 1080p template, all with 3 different output folders. Then all 3 resolutions will go into different folders.

**Reply by mr-qansuh** [reply]
how can i make 3 different output folders. 
can you explain ?
